accepted/PSR-11-container-meta.md

```diff
@@ -226,7 +226,7 @@ As a result, the `ContainerInterface` contains two methods:
 
 While `ContainerInterface` only defines one mandatory parameter in `get()`, it is not incompatible with
 existing containers that have additional optional parameters. PHP allows an implementation to offer more parameters
-as long as they are optional, because the implementation *does* satisfy the interface.
+as long as they are optional because the implementation *does* satisfy the interface.
```

Comment:
I don't think this change is necessary.  It reads better to me with the comma.  (I think this is one of those cases where either way is technically language-legal, because English.)

Reply:
+1 on this from me.

Reply:
@Crell, @Jean85 I reverted this change back as I used to be. I still think it doesn't read that well though, a better solution would be to make it as follows:
```
While `ContainerInterface` only defines one mandatory parameter in `get()`, it is not incompatible 
with existing containers that have additional optional parameters. PHP allows an implementation to offer more parameters 
as long as they are optional (because the implementation *does* satisfy the interface).
```